BraintreeCore/build.gradle

```diff
@@ -82,6 +82,7 @@ dependencies {
     testImplementation project(':TestUtils')
     testImplementation project(':UnionPay')
     testImplementation project(':Venmo')
+    testImplementation("org.jetbrains.kotlin:kotlin-test:1.8.0")
```

Comment:
Can we put this in the top-level `build.gradle` deps section (open to other ways to centralize dependencies, but that's currently the pattern we follow)? 

Also curious if the existing `kotlin-test-junit` library could be used or if this is preferred?

Reply:
+1, ideally we can leverage one of the dependencies we already have imported

Reply:
It looks like `kotlin-test-junit` builds on top of `kotlin-test`.
I was able to use the existing dependency and it still worked.